scripts/exgefs_wave_stat_pnt.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+#                                                                       
+################################################################################
+#                                                                              #
+# exgefs_stats_pnt.sh - Compute point output statistics for gefs wave products #
+#                                                                              #        
+# Packs ensemble mean, spread and probabilities in bull_tar and station_tar    #
+#                                                                              #             
+# Requirements:                                                                #    
+# - WGRIB2 with IPOLATES library                                               #              
+#                                                                              #              
+# Origination:                                                                 #
+# - EMC Wave model Developer- Saeideh Banihashemi, March 2025 	               #               
+#                                                                              #          
+# Update log since 2025                                                        #
+# Jan2025 SBanihash - Adding this script to the global workflow                #
+################################################################################
+#
+cat << EOF
+
+    *******************************************
+    ***WAVE POINT JOB ENSEMBLE STATS SCRIPT****
+    *******************************************
+
+Starting at : $(date)
+-------------
+
+EOF
+
+#
+# 0.a Define model grid
+#
+source "${USHgfs}/wave_domain_grid.sh"
+process_grdID "${wavepostGRD}"
+
+# Script will run only if pre-defined NTASKS
+#     The actual work is distributed over these tasks.
+#
+#
+if [[ -z "${NTASKS}" ]]; then
+  export err=1
+  err_exit "Requires NTASKS to be set"
+fi
+MEMDIR="ensstat"  YMD=${PDY} HH=${cyc} declare_from_tmpl COMOUT_WAVE_STATION_ENS:COM_WAVE_STATION_TMPL
+
+#
+# 0.b Time management
+#
+#fhr3=$(printf %03i ${FORECAST_HOUR})
+valid_time=$(date -u -d "${PDY} ${cyc}" "+%Y%m%d%H")
+ymdh_init=$(date -u -d "${valid_time:0:8} ${valid_time:8:2} - ${WAVHINDH} hours" "+%Y%m%d%H")
+
+mkdir output_${ymdh_init}
+cd output_${ymdh_init} || exit
+
+STATION_TAR="./${RUN}.t${cyc}z.station_tar"
+BULL_TAR="./${RUN}.t${cyc}z.bull_tar"
+
+
+
+# 1.a Check if buoy input files exist and copy
+dir_var="${COMOUT_WAVE_STATION_ENS}"
+
+# Ensure directory exists before proceeding
+if  [ ! -d "$dir_var" ]; then
+  echo "Error: Directory '$cpdir' does not exist or is empty."
+  exit 2
+fi
+# Use ls to safely check for matching files
+for file in "${dir_var}/${RUN}.t${cyc}z.f"???.*_tar; do
+  if [[ -f "$file" ]]; then  # Ensure it's a file before linking
+    cp -rp "$file" .
+    #this line needs to be figured out when we decide where these temp files are saved,
```

Comment:
Please place a `TODO:` to mark as something that needs to be done.  Open an issue if neccesary to remind